README.md

```diff
@@ -1,206 +1,23 @@
-# Microsoft Power Platform Connectors
+# Title
```

Comment:
@jforscht thank you so much for fixing your files.
Only a small change is required here in the readme file. 
You have the following.
```
# Title
Impower is an ERP system for property managers. This connector provides access to the Open API of Impower to send information to the ERP system as well as get information from the ERP​.

## Publisher: Publisher's Name
Impower GmbH ​

## Prerequisites
A license to Impower is required to use this connector. For details please visit www.impower.de​

## Supported Operations
The following actions are supported by the connector.​
### Upload Invoice
With this operations a pdf file is uploaded to Impower as new invoice.

### Get invoice details
With this operations booking details of an invoice will be provided.

## Obtaining Credentials
Credentials to the API will be provided by the Impower support team.​
support@impower.de

## Known Issues and Limitations
Currently there are no known issues with this connector.
```

Please change to 
```
# Impower
Impower is an ERP system for property managers. This connector provides access to the Open API of Impower to send information to the ERP system as well as get information from the ERP​.

## Publisher: Impower GmbH ​

## Prerequisites
A license to Impower is required to use this connector. For details please visit www.impower.de​

## Supported Operations
The following actions are supported by the connector.​
### Upload Invoice
With this operations a pdf file is uploaded to Impower as new invoice.

### Get invoice details
With this operations booking details of an invoice will be provided.

## Obtaining Credentials
Credentials to the API will be provided by the Impower support team.​
support@impower.de

## Known Issues and Limitations
Currently there are no known issues with this connector.
```